src/views/List.jsx

```diff
@@ -32,9 +32,16 @@ export function List({ data }) {
 					X
 				</button>
 			)}
+			<p>Mark your purchases, they will automatically uncheck after 24hrs.</p>
```

Comment:
This is completely a nitpick and potentially a question for our mentors but I want to know what the point of this functionality is. It is still not clear from the code. Is it so that I buy the item, check it, as i purchased it, and then after 24 hours it unchecks so I can but it again? Will the dateNextPurchased come into play later to make this more clear?

Reply:
Yeah thats what I currently think. A user would create a List, then add items to the list. Then they go shopping and check off the items as they purchase them. After 24hrs it'll uncheck. 

The code currently handles manual checking-off as an accident or the user couldn't purchase the item in the end.

We didn't think about dateNextPurchase as the issue mentioned not to.

Reply:
Yes I saw the issue mentioned not to focus on dateNextPurchased, your comments have helped me understand this better! I'm going to do a little re-review of the code and app to further my understanding. I appreciate all your clarifications! :3